_drafts/SC3-1-1-html-has-lang.md

```diff
@@ -0,0 +1,42 @@
+---
+name: SC3-1-1-html-has-lang
+
+description:
+  This rule checks that the `html` element has a `lang` or `xml:lang` attribute.
+
+success_criterion:
+- 3.1.1
+
+test_aspects:
+- DOM Tree # The tree that HTML is parsed into.
+
+authors:
+- Annika Nietzio
+- Jey Nandakumar
+---
+
+## Test Procedure
+
+### Applicability
+
+The root node of the page, if it is an `html` element.
```

Comment:
Same comment as this: https://github.com/auto-wcag/auto-wcag/pull/80#discussion_r181698152

Reply:
Cool, changed to root element.